src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/customerSegment-property-values.js

```diff
@@ -1,9 +1,26 @@
-angular.module('virtoCommerce.demoSolutionFeaturesModule')
-.controller('virtoCommerce.demoSolutionFeaturesModule.customerSegmentPropertyValuesController', ['$scope', 'platformWebApp.bladeNavigationService', 'platformWebApp.dynamicProperties.dictionaryItemsApi', function ($scope, bladeNavigationService, dictionaryItemsApi) {
+angular.module('virtoCommerce.DemoSolutionFeaturesModule')
+.controller('virtoCommerce.DemoSolutionFeaturesModule.customerSegmentPropertyValuesController', ['$scope', 'platformWebApp.bladeNavigationService', 'platformWebApp.dynamicProperties.dictionaryItemsApi', function ($scope, bladeNavigationService, dictionaryItemsApi) {
     var blade = $scope.blade;
     blade.toolbarCommands = [
         {
-            name: "platform.commands.preview", icon: 'fa fa-eye'
+            name: "platform.commands.preview", icon: 'fa fa-eye',
+            executeMethod: (currentBlade) => {
+                let properties = currentBlade.currentEntities
+                    .filter(property => property.valueType === 'Boolean' || property.values.length && property.values.every(value => !!value.value))
+                    .reduce((propertiesAggregate, property) => {
+                        propertiesAggregate[property.name] = property.values && property.values.length ? property.values.map(propertyValue => propertyValue.value.name || propertyValue.value) : [false];
```

Comment:
Please add a short comment why we doing so.

Reply:
Done